### PR TITLE
[VPN] Use local provisionprofile instead of URL

### PR DESCRIPTION
**Pull request by hneiva**

### CODE REVIEW

File: iscript/src/iscript/macvpn.py [hunk 1]
```diff
@@ -60,12 +59,19 @@ async def _sign_app(config, sign_config, app, entitlements_url, provisioning_pro
...
         entitlements_url (str): URL for entitlements file
-        provisioning_profile_url (str): URL for provisioning profile
+        provisionprofile_filename (str): .provisionprofile filename in <config.provisionprofile_folder>
 
     """
     # We mock the task for downloading entitlements and provisioning profiles
     entitlements_path = await download_entitlements_file(config, sign_config, {"payload": {"entitlements-url": entitlements_url}})
-    provisioning_profile_path = await download_provisioning_profile(config, {"payload": {"provisioning-profile-url": provisioning_profile_url}})
+    provisioning_profile_path = None
+    if config.get("provisionprofile_folder"):
```
Comment (ahal) [inline iscript/src/iscript/macvpn.py]: nit: `provisionprofile_dir`?

File: iscript/src/iscript/macvpn.py [hunk 2]
```diff
@@ -60,12 +59,19 @@ async def _sign_app(config, sign_config, app, entitlements_url, provisioning_pro
...
+        provisioning_profile_path = os.path.join(config.get("provisionprofile_folder"), provisionprofile_filename)
```
Comment (ahal) [inline iscript/src/iscript/macvpn.py]: nit: don't need to use `config.get` here

**Review by ahal — CHANGES_REQUESTED** [review]
I think we should remove the `download_provisioning_profile` function as well? Or do you think we'll use that for other projects?

Otherwise LGTM!

File: iscript/src/iscript/macvpn.py [hunk 3]
```diff
@@ -60,12 +59,19 @@ async def _sign_app(config, sign_config, app, entitlements_url, provisioning_pro
...
+        if not os.path.exists(provisioning_profile_path):
+            log.error(f"Could not find provisionprofile file: {provisioning_profile_path}")
+            raise IScriptError("Something")
```
Comment (ahal) [inline iscript/src/iscript/macvpn.py]: Did you mean to update this to something more informative? :)

**Review by ahal — APPROVED** [review]
Thanks!

**Comment by hneiva** [comment]
> I think we should remove the `download_provisioning_profile` function as well? Or do you think we'll use that for other projects?
> 
> Otherwise LGTM!

I'll write a ticket for it so we can check with gecko/other products.

**Pull request merged**